cluster/gce/gci/configure-helper.sh

```diff
@@ -1715,6 +1731,67 @@ function start-kube-apiserver {
   sed -i -e "s@{{admission_controller_config_volume}}@${admission_controller_config_volume}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_mount}}@${image_policy_webhook_config_mount}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_volume}}@${image_policy_webhook_config_volume}@g" "${src_file}"
+
+  if [[ -z ${ETCD_KMS_KEY_ID:-} ]]; then
+    # Removing KMS related placeholders.
+    sed -i -e " {
+        s@{{kms_plugin_container}}@@
+
+        s@{{kms_socket_mount}}@@
+        s@{{encryption_provider_mount}}@@
+
+        s@{{kms_socket_volume}}@@
+        s@{{encryption_provider_volume}}@@
+    } " "${src_file}"
+  else
+    local kms_plugin_src_file="${src_dir}/kms-plugin-container.manifest"
+
+    if [[ ! -f ${kms_plugin_src_file} ]]; then
+        echo "Error: KMS Integration was requested, but ${src_dir}/kms-plugin-container.manifest is missing."
+        exit 1
+    fi
+
+    if [[ ! -f ${encryption_provider_config_path} ]]; then
```

Comment:
wrap variable expansion in weak quotes i.e. `"${encryption_provider_config_path}"`

there are a couple other places in this cl that need this as well.